.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "6"
```

Comment:
@ananyo2012 Looking at the docs, I realized that it is infact possible to use the latest version and not a locked to a specific version. https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions for more. Thanks.